File: hw/carvajal/u2/hw19Inheritance/JSFarmSystem/model/Pig.js
import { FarmAnimal } from "./FarmAnimal.js";

export class Pig extends FarmAnimal {
    constructor(id, breed, bornOn, gender, isAbleToReproduce, weight, cage) {
        super(id, breed, bornOn, gender, isAbleToReproduce, weight, cage);
    }

    toString() {
        return `Pig{${super.toString()}}`;
    }
}

File: hw/chalan/u2/hw19Inheritance/Farmsystem/src/main/java/ec/edu/espe/farmsystem/model/Pig.java
package ec.edu.espe.farmsystem.model;

import java.util.Date;

/**
 * @author Kevin Chalan, Object Masters, @ESPE
 */
public class Pig extends FarmAnimal {

    public Pig(int id, String breed, Date bornOn, String gender, boolean isAbleToReproduce, float weight, Cage cage) {
        super(id, breed, bornOn, gender, isAbleToReproduce, weight, cage);
    }
    
    @Override
    public String toString() {
        return "Pig{" + super.toString() + super.toString()+'}';
    }
}